AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/src/ErrorMessages.dfy

```diff
@@ -57,6 +57,9 @@ module {:options "/functionSyntax:4" } KeyStoreErrorMessages {
   const INVALID_HIERARCHY_VERSION :=
     "Invalid hierarchy version. Expected version 1 or 2."
 
+  const BRANCH_KEY_MD_DIGEST_SHA_INCORRECT_LENGTH :=
+    "Branch key + md digest sha is of incorrect length."
```

Comment:
Blocking: No customer will know what this means. 
Look at this comment on made on Rishav's PR:
https://github.com/aws/aws-cryptographic-material-providers-library/pull/1342#discussion_r2004761345

Reply:
https://github.com/aws/aws-cryptographic-material-providers-library/pull/1354#discussion_r2011018197